resteasy/src/test/java/com/cerner/beadledom/resteasy/TestProvider.java

```diff
@@ -0,0 +1,20 @@
+package com.cerner.beadledom.resteasy;
+
+import javax.inject.Provider;
+
+/**
+ * Provides a fully-constructed instance of {@link TestJaxRsProvider}.
+ */
+public class TestProvider implements Provider<TestJaxRsProvider> {
+
+    /**
+     * Constructor for TestProvider.
```

Comment:
This comment isn't helpful.

Reply:
https://github.com/cerner/beadledom/pull/143/commits/75d364f068be5b48f12a864aa2a8bd938fb929b8